Body.cpp

```diff
@@ -5,36 +5,37 @@
 #include "Body.h"
 
 glm::vec3 Body::GetCenterOfMassWorldSpace() const {
-    glm::vec3 centerOfMass = m_Shape->GetCenterOfMass();
-    glm::mat3 r = glm::mat3_cast(m_Orientation);
-    centerOfMass = r * centerOfMass;
-    return m_Position + centerOfMass;
+	glm::vec3 centerOfMass = m_Shape->GetCenterOfMass();
+	glm::mat3 r = glm::mat3_cast(m_Orientation);
+	centerOfMass = r * centerOfMass;
+	return m_Position + centerOfMass;
 }
 
 glm::vec3 Body::GetCenterOfMassModelSpace() const {
-    return m_Shape->GetCenterOfMass();
+	return m_Shape->GetCenterOfMass();
 }
 
-glm::vec3 Body::WorldSpaceToBodySpace(glm::vec3 &pt) {
-    glm::vec3 tmp = pt - GetCenterOfMassWorldSpace();
+glm::vec3 Body::WorldSpaceToBodySpace(glm::vec3 &pt) const {
+	glm::vec3 tmp = pt - GetCenterOfMassWorldSpace();
 
-    // m_Orientation is the rotation in WorldSpace.
-    // So we need to rotate back to get the original pt in BodySpace
-    glm::quat inverseOrient = inverse(m_Orientation);
+	// m_Orientation is the rotation in WorldSpace.
+	// So we need to rotate back to get the original pt in BodySpace
+	glm::quat inverseOrient = inverse(m_Orientation);
 
-    glm::mat3 r = glm::mat3_cast(inverseOrient);
-    glm::vec3 bodySpace = r * tmp;
-    return bodySpace;
+	glm::mat3 r = glm::mat3_cast(inverseOrient);
+	glm::vec3 bodySpace = r * tmp;
+	return bodySpace;
 }
 
-glm::vec3 Body::BodySpaceToWorldSpace(glm::vec3 &pt) {
-    glm::mat3 r = glm::mat3_cast(m_Orientation);
-    glm::vec3 ptWorld = r * pt;
-    return GetCenterOfMassWorldSpace() + ptWorld;
+glm::vec3 Body::BodySpaceToWorldSpace(glm::vec3 &pt) const {
+	glm::mat3 r = glm::mat3_cast(m_Orientation);
+	glm::vec3 ptWorld = r * pt;
+	return GetCenterOfMassWorldSpace() + ptWorld;
 }
 
 void Body::ApplyImpulseLinear(glm::vec3 &impulse) {
-    if (m_InvMass <= 0.0f) return;
+	if (m_InvMass <= 0.0f)
+		return;
```

Comment:
we may want to add braces here. this should also be doable over an automation

Reply:
Yes please! Are the current rules disallowing one-line `if` statements?
I think those can be nice sometimes, especially for early returns. But multi-line `if` blocks should *always* have braces.

Reply:
i would have to look. but i always write code with the paradigm: "optimize for change" those few lines add a lot of readability and lets you change the inside of the if clause easily